doc/tech/glossary.md

```diff
@@ -108,10 +108,16 @@ Terms not defined by Kubernetes:
 - **CaaS**:	Container-as-a-Service. A complete set of technologies to enable the management of containerised software, including a Kubernetes cluster, container networking, storage, routing, service mesh, etc.
 - **CaaS Manager**:	A management plane function that manages the lifecycle (instantiation, scaling, healing, etc.) of one or more CaaS instances, including communication with VIM for master/node lifecycle management.
 
-<a name="11.1.5"></a>
-### 11.1.5 Other Referenced Terminology
+<a name="1.5"></a>
+## 1.5 Other Referenced Terminology
 
 - **Virtualised Infrastructure Manager (VIM)**: responsible for controlling and managing the **NFVI** compute, storage and network resources.
 - **NFV Orchestrator (NFVO)**: manages the VNF lifecycle and **NFVI** resources (supported by the **VIM**) to ensure an optimised allocation of the necessary resources and connectivity.
```

Comment:
Suggest changing `NFVI` to `Cloud Infrastructure` to cover all infrastructure types.

Reply:
done: https://github.com/cntt-n/CNTT/pull/1267/commits/54f098514759bee6ec0a5a0ab59716b96112235b

Reply:
done